docs/index.adoc

```diff
@@ -61,8 +61,13 @@ How it works: annotate your class and picocli initializes it from the command li
 
 TIP: Picocli also provides a <<Programmatic API,programmatic API>>, separately from the annotations API.
 
-=== Example application
-The example below shows a short but fully functional picocli-based `checksum` command line application:
+=== Example application: Checksum
+`Checksum` is a short but fully functional picocli-based command line application which prints the checksum (MD5 by default) of a file to STDOUT.
+
+[TIP]
+====
+You might want to run the `checksum` application online first. To do so, open the source code in a separate https://www.jdoodle.com/embed/v0/2mQM?stdin=1&arg=1[browser tab^] and press the blue `Execute` button below. This prints the checksum of the file `/etc/hosts` to the console. Then type a file name (e.g. `/usr/bin/java` or `/proc/cpuinfo`) into the text field `Commandline Arguments` and execute the program again. Fill in `--help` to get usage instructions.
+====
```

Comment:
Great! I like it.

Reply:
One thing: I like the convention of one sentence per line in AsciiDoc.
Can you change that?

Reply:
AsciiDoc now has on line per sentence.
As another minor change, I changed the link text.